app/processing/replace-sfi22.js

```diff
@@ -0,0 +1,21 @@
+const replaceSFI22 = (dbExtract) => {
+  for (let i = 0; i < dbExtract.length; i++) {
+    if (dbExtract[i].name === 'SFI') {
+      dbExtract[i].name = 'SFI22'
+    }
+    if (dbExtract[i].schemes?.name === 'SFI') {
+      dbExtract[i].schemes.name = 'SFI22'
+    }
+    if (dbExtract[i].ledgerPaymentRequest?.schemes?.name === 'SFI') {
+      dbExtract[i].ledgerPaymentRequest.schemes.name = 'SFI22'
+    }
+    if (dbExtract[i].paymentRequest?.schemes?.name === 'SFI') {
+      dbExtract[i].paymentRequest.schemes.name = 'SFI22'
+    }
+  }
```

Comment:
After seeing this, feels it's not really the way to go.

This now needs to have knowledge of data structures from every calling function.  If it can't be made generic without the if statements then maybe we shouldn't do this.

Reply:
refactor undone in that case!